### ISSUE TITLE
How to specify multiple owner in family_owners_list for a family of lists

### ISSUE THREAD

**Issue by jth56000** [sympa-community/sympa#786]
Hello,

On our 6.2.22 SYMPA server we use family of lists.
I would like for one family in robot.conf file to specify multiple users in family_owners_list but it does not work.

How can I manage to do this so that multiple user can create lists in this family ?

Thank you

**Comment by jth56000** [comment]
Hello,
We recently migrate our server in SYMPA 6.2.52 but we still have this issue.
Can anybody help us ?

Thank you
Jean

**Comment by ikedas** [comment]
Hi @jth56000 ,

I think you may add multiple users to the list specified by `family_owners_list` sub-parameter.

**Comment by jth56000** [comment]
Hello Soji,

 

It does not work, if I set 2 users to family_owners_list thez do not see the family in the top menu and they can’t create a list.

 

Jean

 

Hi @jth56000 <https://github.com/jth56000>  ,

I think you may add multiple users to the list specified by family_owners_list sub-parameter.

**Comment by ikedas** [comment]
> Hello Soji,
> It does not work, if I set 2 users to family_owners_list thez do not see the family in the top menu and they can’t create a list.

I think you mean you have done following:

1. You created a list named, say, `ownerlist@dom.ain`.
2. You included a subparameter `family_owners_list=ownerlist` in `automatic_list_families` parameter.
3. You added two subscriber, say, `user-a@some.dom.ain` and `user-b@some.dom.ain` to `ownerlist` list.

Is it right?

**Comment by jth56000** [comment]
I create a family of list in automatic_list_families of my robot.conf with the appropriates files in /etc/sympa/<my domain>/families/<my family>

 

I set 2 owners separate by , in family_owners_list for my family of list

 

You mean I should create a specific list of owners for family_owners_list which cannot be a list of , separated owners.

 

Jean

Hello Soji,
It does not work, if I set 2 users to family_owners_list thez do not see the family in the top menu and they can’t create a list.

I think you mean you have done following:

1.	You created a list named, say, ownerlist@dom.ain <mailto:ownerlist@dom.ain> .
2.	You included a subparameter family_owners_list=ownerlist in automatic_list_families parameter.
3.	You added two subscriber, say, user-a@some.dom.ain <mailto:user-a@some.dom.ain>  and user-b@some.dom.ain <mailto:user-b@some.dom.ain>  to ownerlist list.

Is it right?

**Comment by ikedas** [comment]
> You mean I should create a specific list of owners for family_owners_list which cannot be a list of , separated owners.

Yes! Subscriber(s) of the list that is specified by `family_owners_list` sub-parameter are recognized as "family owners".

**Comment by jth56000** [comment]
Ok It works, thank you for your help.

 

De : IKEDA Soji <notifications@github.com> 
Envoyé : mercredi 24 juin 2020 11:09
À : sympa-community/sympa <sympa@noreply.github.com>
Cc : THOMAS Jean <jean.thomas@ext.dsi.cnrs.fr>; Mention <mention@noreply.github.com>
Objet : Re: [sympa-community/sympa] How to specify multiple owner in family_owners_list for a family of lists (#786)

 

You mean I should create a specific list of owners for family_owners_list which cannot be a list of , separated owners.

Yes! Subscriber(s) of the list that is specified by family_owners_list sub-parameter are recognized as "family owners".

**Comment by ikedas** [comment]
Thanks for sharing your experiences!

**Comment by jth56000** [comment]
It looks like it does not work so good, I added a list to family_owners_lis but users in that list do not have the family list creation menu in top menu.

**Comment by ikedas** [comment]
> It looks like it does not work so good, I added a list to family_owners_lis but users in that list do not have the family list creation menu in top menu.

@jth56000 , please show us the configuration.

**Comment by jth56000** [comment]
Hello Soji,



Sorry to disturb you with this issue again.

In my robot.conf I have :



name=jean-test:prefix=jean-test:display=Listes Test Jean:prefix_separator=+:classes_separator=:family_owners_list=listownertest@listes-int.dsi.cnrs.fr



Where listownertest@listes-int.dsi.cnrs.fr  is a list with 3 users.



What else do you need ?



Jean

   It looks like it does not work so good, I added a list to family_owners_lis but users in that list do not have the family list creation menu in top menu.

   @jth56000<https://github.com/jth56000> , please show us the configuration.
